sympy/control/core.py

```diff
@@ -0,0 +1,269 @@
+"""
+A symbolic control toolbox.
+"""
+
+import sympy
+import mpmath
+
+# pylint: disable=invalid-name
+
+s, t = sympy.symbols('s, t')
+
+def array_to_poly(a, s):
```

Comment:
This should operate on an iterable for `a`. Also, can't the Poly class we already have to this? `array` isn't an appropriate name unless it actually uses the SymPy array object.